src/core_plugins/kibana/public/assets/play-circle.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="8" height="8" viewBox="0 0 8 8">
+  <path d="M4 0c-2.21 0-4 1.79-4 4s1.79 4 4 4 4-1.79 4-4-1.79-4-4-4zm-1 2l3 2-3 2v-4z" />
+</svg>
```

Comment:
It looks to me like they do get copied over, but the files are renamed with a long hash. The image probably needs to be imported so webpack has a path to follow. I'm guessing the existing icons in src/ui/public/icons get included because of this less file https://github.com/Bargs/kibana/blob/46c3eaca6e80859b42964ebaf11f77a7069b233a/src/ui/public/styles/base.less#L640-L640